.github/workflows/ci.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
           path: target
           key: ${{ runner.os }}-cargo-build-target-${{ hashFiles('**/Cargo.lock') }}-${{ matrix.rust_version }}
       - name: Run tests
-        run: cargo test --verbose
+        run: cargo test --verbose --all --no-fail-fast --all-features
```

Comment:
I'm not sure about the `--no-fail-fast` and `--all-features` though.

Reply:
that's fine for now